components/button/button.tsx

```diff
@@ -190,21 +193,30 @@ const InternalCompoundedButton = React.forwardRef<
     return cleanupTimer;
   }, [loadingOrDelay]);
 
+  // Two chinese characters check
   useEffect(() => {
     // FIXME: for HOC usage like <FormatMessage />
-    if (!buttonRef || !(buttonRef as any).current || !mergedInsertSpace) {
+    if (!buttonRef.current || !mergedInsertSpace) {
       return;
     }
-    const buttonText = (buttonRef as any).current.textContent;
+    const buttonText = buttonRef.current.textContent || '';
     if (needInserted && isTwoCNChar(buttonText)) {
```

Comment:
regex `/^[\u4E00-\u9FA5]{2}$/` in `isTwoCNChar()` is not compatible with `Unicode 4.1+`, which was released at 2005
ref: https://www.unicode.org/charts/PDF/Unicode-4.1/U41-4E00.pdf